utils/common/src/main/kotlin/ProcessCapture.kt

```diff
@@ -129,9 +129,7 @@ class ProcessCapture(
         }
 
     init {
-        logger.info {
-            "Running '$commandLine' in '$usedWorkingDir'..."
-        }
+        logger.debug { "Running '$commandLine' in '$usedWorkingDir'..." }
```

Comment:
I find this to be one of the most useful log lines when debugging failed runs, I wouldn't want it to appear only in debug.

Reply:
Ok, I've dropped it.